ATMoviess/ATMoviess/Models/UpcomingMoviesModel.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace ATMoviess.Models
+{
+    public class UpcomingMoviesModel
+    {
+        public List<Result> Results { get; set; }
+        public int Page { get; set; }
+        public int Total_results { get; set; }
```

Comment:
You can use a `JsonAttribute` or `SnakeCaseContractResolver` so you don't have to mess with your models to match the API specs

Reply:
Good idea! Thanks for the feedback!